gce/image/files/scylla_install_image

```diff
@@ -105,7 +105,7 @@ if __name__ == '__main__':
     run('systemctl daemon-reload')
     run('systemctl enable scylla-image-setup.service')
     run('/opt/scylladb/scripts/scylla_setup --no-coredump-setup --no-sysconfig-setup --no-raid-setup --no-io-setup --no-bootparam-setup --no-ec2-check --no-swap-setup')
-    run('/opt/scylladb/scripts/scylla_sysconfig_setup --ami')
+    run('/opt/scylladb/scripts/scylla_sysconfig_setup --ami --disable-writeback-cache')
```

Comment:
Shouldn't we remove the `--ami` flag?

Reply:
@yaronkaikov Good question, but not for this pull request to address.